test_dir_shadowing/test_dir_shadowing_main_PY2_filelist.txt

```diff
@@ -9,3 +9,4 @@ subpar/test_dir_shadowing/test_dir_shadowing/dir_shadowing_lib_dat.txt
 subpar/test_dir_shadowing/test_dir_shadowing/dir_shadowing_main.py
 subpar/test_dir_shadowing/test_dir_shadowing/dir_shadowing_main_dat.txt
 subpar/test_dir_shadowing/test_dir_shadowing_main
+UNPAR_MANIFEST
```

Comment:
do you think we should only add the `UNPAR_MANIFEST` if `extract_dir` is set?

Reply:
Good point, it's pretty much completely extraneous in the non-unpar case. I can make that change along with the next PR